Automattic/woocommerce-payments
# Fix variable virtual products require a shipping address for Apple Pay and Google Pay

Fixes #6714

#### Changes proposed in this Pull Request
Fixes the issue where Apple Pay and Google Pay require a shipping address for variable virtual products.

<img width="424" alt="Screenshot 2023-09-28 at 13 04 08" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/94986c8a-0e93-4607-923c-e4c13d58a9fd">

#### Testing instructions
- Create a variable product, and tick "virtual" checkboxes in all variations.
- On the product page, select a virtual variation and click "Buy with Apple Pay".
- Verify a shipping address is not required on the modal.

#### Additional context
p1695730786442689-slack-C9976E5MJ

-------------------


**Post merge**
